frontend/lib/components/drawer.dart

```diff
@@ -0,0 +1,54 @@
+import 'package:flutter/material.dart';
+import 'package:frontend/pages/home.dart';
+import 'package:frontend/pages/calendar.dart';
+import 'package:frontend/pages/user_events.dart';
+
+class AppDrawer extends StatelessWidget {
```

Comment:
Leaving some in-line comments here as well would be nice for the readibility for the students